src/Datadog.Trace.ClrProfiler.Native/integration_loader.h

```diff
@@ -17,17 +17,31 @@ using json = nlohmann::json;
 
 // LoadIntegrationsFromEnvironment loads integrations from any files specified
 // in the DD_INTEGRATIONS environment variable
-std::vector<Integration> LoadIntegrationsFromEnvironment();
+void LoadIntegrationsFromEnvironment(std::vector<IntegrationMethod>& integrationMethods, const bool isCallTargetEnabled,
+                                     const bool isNetstandardEnabled,
+                                     const std::vector<WSTRING> disabledIntegrationNames);
```

Comment:
maybe pass the `vector` disabledIntegrationNames by `const &` instead

Reply:
done in 1aa6b9d